system_strings/sysmes_text.en

```diff
@@ -1230,32 +1230,32 @@ I leave the sleeping Arcueid and^exit the apartment alone, heading^for the depar
-映画が定番だろう。面白みが^ないとか言わないでほしい。^定番こそ王道である。
-我ながら正気だろうか。^だがこの選択に間違いはない。^大いなる野望を実現させるのだ。
-さいわい、今日は天気もいい。^アルクェイドを連れて、^日中の公園へとやってきた。
-彼女の話。^食べ物の話。^その、敵の話。
-アルクェイドにせがまれて、^自分の通う学校へとやってきた。^一体どういうつもりなのか……。
-彼女とした、^“もしも”のはなし。
-アルクェイドと公園に来た。^探索を始めるには、まだ早い。^ただ、時間が過ぎるのを待つ。
-試そうか、と彼女は言った。^その吐息が首筋に迫る。^言い知れぬ恐怖に、自分は――
-悪ふざけ、で良かったのに。^アルクェイドは謝罪を口にして、^目の前から跳び去った。
-聖堂教会の司祭は語る。^アルクェイドの“敵”の正体と^その目的を。
-もう余計な無駄道は踏めない。^アルクェイドを追うため、^夜の街へと走り出した。
-アルクェイドは見つからない。^あいつがいるとすれば――
-彼女の部屋に行こう。^今夜は死者を探す事も諦めて、^部屋に戻っているかもしれない。
-部屋に戻ったとは思えない。^向かうのなら、今は誰も^やってこない場所―――
-眼鏡を外し、夜の街を行く。^彼女の死だけは視えない。^それが手掛かりになると信じて。
-夜の学校には人気がない。^アルクェイドはもしかしたら、^ここに来ているのかもしれない。
-介錯の牙が別れを告げる。^欲望は、^宿命をより強く補強した。
-アルクェイドを泣かせるような^真似は、二度とできない。^この脳が焼き切れても―――
+The end always has a reason. Be^that as it may, those who do not^know their own pasts have no way^of knowing.
+I awaken to a refreshing^morning. ―――Wait, what the hell^is this?
+I'm to blame for keeping Arcueid^waiting all night. Now that it's^come to this, I'll have to spend^the whole day with her for^today.
+Why were you so angry? Well,^that's because you're selfish...^is that why?
+Well, isn't it because you want^revenge on me for killing you...
+To be honest, I'd like to ask^that myself. Because you're a^vampire... or something?
+First, let's leave the mansion.^There's nowhere to enjoy^ourselves here.
+In order to entertain Arcueid,^we head to the direction of the^station... Wait, what is this^princess doing?
+A movie is standard fare. Don't^call it boring or something like^that. It is exactly because it's^standard that it's the^tried-and-true method.
+I wonder if I'm even sane. But^there's no doubt about it, I^must fulfill my overwhelming^desire.
+Fortunately, today's weather^is great. I decide to bring her^along to the park.
+Talking about her. Talking^about food. Talking about the^enemy.
+After being pestered by Arcueid,^we ended up at my school. What^on earth does she plan to do...
+Talking about 'what-ifs'.
+I arrived at the park, together^with Arcueid. It's too early to^begin our search, so we wait as^time passes by.
+'Test it out', she said. Her^breath approaches my nape. In^the face of this inexplicable^fear, I――
+Even though it was supposed to^be a joke. Arcueid apologizes^and leaps away.
+The Holy Church's Presbyter^speaks. About her 'enemy's'^true form, and his purpose.
+I can't afford to waste any more^time. I rush into the nighttime^city to chase after Arcueid.
+I can't find Arcueid. If she was^here, she'd be――
+I'll go to her room. She^might've given up searching for^the Dead for tonight, and gone^back to her room.
+I don't think she went back^to her room. If she was to go^somewhere, it'd be somewhere no^one would go to―――
+I remove my glasses and head^into the nighttime city. She's^the only one whose death I can't^see. I'll believe that'll help^me find her.
+There isn't anyone at school^during nighttime. There's a^possibility Arcueid could've^gone there.
+The fangs of beheading bid^farewell. My desires reinforced^my fate even further.
```

Comment:
petrikow said this refers to the assisted beheading of seppuku, but i dont really have any alternatives that fit